File: Starter Problems 1/mix.py
op = open('mixin.txt', 'r')
a, b = map(int, op.readline().split())
c = a / b
a = a % b

out = open('mixout.txt', 'w')
if a != 0:
	out.write(str(c) + ' ' + str(a) + '/' + str(b))
else:
	out.write(str(c))

File: Starter Problems 1/mix.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ifstream inFile;
    inFile.open("mixin.txt");
    int a,b;
    inFile >> a;
    inFile >> b;
    inFile.close();
    ofstream outFile;
    outFile.open("mixout.txt");
    if (a % b == 0) {
        outFile << a/b;
    }
    else {
        int remainder = a - int(a/b)*b;
        outFile << int(a/b) << " " << remainder << "/" << b;
    }
    outFile.close();
}
